src/Wig.Zstd.Tests/DummyFacts.cs

```diff
@@ -0,0 +1,20 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="DummyFacts.cs" company="Simply Effective Solutions">
+//   Copyright (c) 2008 - GFG Simply Effective Solutions. All rights reserved.
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+
+namespace Wig.Zstd.Tests
+{
+    using NUnit.Framework;
+
+    [TestFixture]
+    public class DummyFacts
+    {
+        [Test]
+        public void DummyTest()
+        {
```

Comment:
We need to add some real tests. We can discuss at the meeting on Wednesday but would like Dmitry to start thinking about it